lib/rex/ui/text/shell.rb

```diff
@@ -2,6 +2,8 @@
 require 'rex/text/color'
 require 'rex/ui'
 
+gem 'pry-byebug'
+
```

Comment:
I think this was left behind whilst you were debugging this issue, would probably be a good idea to remove this.

Reply:
Good call, forgot about that. Removed.